docs/main/updating/plugins/4-0.md

```diff
@@ -0,0 +1,79 @@
+
+There are only a few changes required to update a plugin from Capacitor 3 to Capacitor 4.
+
+:::info
+While you can follow the manual steps below, check out [this tool](https://github.com/rdlabo-team/capacitor-plugin-to-v4) from community member [@rdlabo](https://twitter.com/rdlabo) that automatically updates a Capacitor 3 plugin to v4.
+:::
+
+## Android
+
+### Update Android Project Variables
+
+In your `build.gradle` file, update your values to the following new minimums:
+
+```groovy
+minSdkVersion = 22
+compileSdkVersion = 32
+targetSdkVersion = 32
+androidxAppCompatVersion = '1.4.2'
+junitVersion = '4.13.2'
+androidxJunitVersion = '1.1.3'
+androidxEspressoCoreVersion = '3.4.0'
+```
+
+### Update to Gradle 7
+
+Adjust your Gradle project settings in `File > Project Structure > Project`. The Android Gradle Plugin Version should be 7.2.1 or later and the Gradle Version should be 7.4.2 or later. Apply these changes and run a gradle sync by clicking on the Elephant Icon in the top right of Android Studio.
+
+:::info
+Android Studio may provide an automatic migration to Gradle 7. Go ahead and take them up on the offer! To upgrade, go to your `build.gradle` file, and click on the 💡 icon, and click "Upgrade Gradle. Once your project is migrated over, run a gradle sync as described above.
+
+Another alternative would be to use the Android Gradle Plugin Upgrade Assistant to handle the migration for you. Steps for this tool can be found in the [Android documentation](https://developer.android.com/studio/build/agp-upgrade-assistant).
+:::
+
+### Ensure you are using Java 11
+
+Capacitor 3 works with both Java 8 and Java 11. Moving forward, Capacitor 4 will only support Java 11. You can change this in your project by going to the following menu in Android Studio:
+
+`Preferences > Build, Execution, Deployment > Build Tools > Gradle`
+
+<img
+  src={require('/img/v4/docs/main/updating/android-java-11.png').default}
+  loading="eager"
+/>
+
+From there, you can modify the "Gradle JDK" to be Java 11.
+
+:::info
+Java 11 ships with the latest version of Android Studio. No additional downloads needed!
+:::
+
+### Remove `jcenter()` from the Gradle files
+
+In previous Capacitor versions, `jcenter()` was required due to our Cordova compatibility layer being hosted on Jcenter. However, we are now using the latest Cordova Android version, hosted on Maven Central. With this, you may be able to remove `jcenter()` entirely from your `build.gradle` file. If you are using other native dependencies, make sure they aren't hosted on Jcenter before removing it!
+
+
+## iOS
+
+### Raise iOS Deployment Target
+
+Do the following to your Xcode project: select the **Project** within the project editor and open the **Build Settings** tab. Under the **Deployment** section, change **iOS Deployment Target** to **iOS 13.0**. Repeat the same steps for any app **Targets**.
+
+Then, open `ios/App/Podfile` and update the iOS version to 13.0:
```

Comment:
```suggestion
Then, open `ios/Podfile` and update the iOS version to 13.0:
```
The guide is about plugins, there is no App folder in plugins, the Podfile is inside the ios folder